editor/plugins/collision_shape_2d_editor_plugin.h

```diff
@@ -52,6 +52,17 @@ class CollisionShape2DEditor : public Control {
 		SEGMENT_SHAPE
 	};
 
+	const Point2 RECT_HANDLES[8] = {
+		Point2(1, 0),
+		Point2(1, 1),
+		Point2(0, 1),
+		Point2(-1, 1),
+		Point2(-1, 0),
+		Point2(-1, -1),
+		Point2(0, -1),
+		Point2(1, -1)
```

Comment:
Add a trailing comma since the array is written over multiple lines:

```suggestion
		Point2(1, -1),
```